flink-runtime-web/web-dashboard/web/css/index.css

```diff
@@ -228,7 +228,6 @@
   overflow: hidden;
   padding: 2px 4px;
   font-size: 14px;
-  -webkit-border-radius: 2px;
```

Comment:
How did this come to be? Are you using a different version of glup or the dependencies to build this?